pkgs/os-specific/linux/kernel/cachyos-kernels.nix

```diff
@@ -0,0 +1,155 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  pkgs,
+  buildLinux,
+  ...
+}@args:
+
+let
+  cachyPatches = fetchFromGitHub {
+    owner = "CachyOS";
+    repo = "kernel-patches";
+    rev = "04fb9178158a7714aeb908c7a5310c4e0d6ea4b6";
+    hash = "sha256-KTvZxYy+UgLNg1DYBMrJrRdxDo3N5ususlrHyRBS7C8=";
+  };
+
+  ltsVariant = {
+    version = "6.6.43";
+    source = {
+      url = "mirror://kernel/linux/kernel/v6.x/linux-6.6.43.tar.xz";
+      hash = "sha256-Ctg7Ghp4ChqtlI1VqlXuY8UMYm8tRpELnSGAAo0QCl4=";
+    };
+    patches = [
+      {
+        name = "cachyos-base-all";
+        patch = "${cachyPatches}/6.6/all/0001-cachyos-base-all.patch";
+      }
+      {
+        name = "bore-cachy";
+        patch = "${cachyPatches}/6.6/sched/0001-bore-cachy.patch";
+      }
+    ];
+
+    config = with lib.kernel; {
+
+    };
+  };
+
+  mainVariant = {
+    version = "6.10.2";
+    source = {
+      url = "mirror://kernel/linux/kernel/v6.x/linux-6.10.2.tar.xz";
+      hash = "sha256-c9hSDdnLpaz8XnII52s12XQLiq44IQqSJOMuxMDSm3A=";
+    };
+    patches = [
+      {
+        name = "cachyos-base-all";
+        patch = "${cachyPatches}/6.10/all/0001-cachyos-base-all.patch";
+      }
+      {
+        name = "sched-ext";
+        patch = "${cachyPatches}/6.10/sched/0001-sched-ext.patch";
+      }
+      {
+        name = "bore-cachy-ext";
+        patch = "${cachyPatches}/6.10/sched/0001-bore-cachy-ext.patch";
+      }
+    ];
+
+    config = with lib.kernel; {
+      # CachyOS config
+      CACHY = yes;
+
+      # CPU Scheduler
+      SCHED_CLASS_EXT = yes;
+      SCHED_BORE = yes;
+      MIN_BASE_SLICE_NS = freeform "1000000";
+
+      # LLVM level
+      LTO_NONE = yes;
+
+      # Tick rate
+      HZ_300 = unset;
+      HZ_1000 = yes;
+      HZ = freeform "1000";
+
+      # Requires override, `lib.mkOverride 60 freeform "320"` didn't work
+      #NR_CPUS = freeform "320";
```

Comment:
Not sure what to do about this

Reply:
> `lib.mkOverride 60 freeform "320"` didn't work

Did you omit the parentheses? `lib.mkOverride 60 (freeform "320")`

Reply:
Yes 😮‍💨